genFigures.py

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env python3
-
+from lineage.figures.figureCommon import overlayCartoon
 import sys
 import time
 import matplotlib
 import numpy as np
 matplotlib.use('AGG')
 
 fdir = './output/'
+cartoon_dir = r"./lineage/figures/cartoons"
```

Comment:
@jclagarde I moved the cartoons to a folder closer to the figures. I didn't really like them in the data folder.